devices/philips.js

```diff
@@ -82,6 +83,26 @@ const fzLocal = {
             return payload;
         },
     },
+
+    philips_hue_multicolor: (opts = {reverse: false}) => {
+        return {
+            cluster: 'manuSpecificPhilips2',
+            type: ['attributeReport', 'readResponse'],
+            convert: (model, msg, publish, options, meta) => {
+                if (msg.data.hasOwnProperty('2')) {
```

Comment:
Should the '2' here be 'multiColor' ?

Reply:
I probably should have commented on it, it's for compatibility until https://github.com/Koenkk/zigbee-herdsman/pull/622 get's merged which maps `2` to `state` (used below).

Reply:
Done!